schemas/README.md

```diff
@@ -4,7 +4,4 @@ Task Cluster JSON Schemas
 In order to substitute in commonly used constants, but have different
 descriptions, and still ensure that it is easy to maintain consistent schemas,
 we have introduce an extra syntax `{"$const": "<key>"}`. These objects will
-be replaced with values from `constants.js` when schemas a rendered.
```

Comment:
IMO, we can remove this file... since it's documented in `base.validator` (I hope)...
before it was sort of magic, and this was included via `require()`...